signing_clients/apps.py

```diff
@@ -385,7 +401,11 @@ def issuer(self):
     def signer_certificate(self):
         for certificate in self.content['certificates']:
             info = certificate['tbs_certificate']
-            if info['issuer'] == self.issuer and info['serial_number'] == self.signer_serial_number:
+            is_signer_certificate = (
+                info['issuer'] == self.issuer and
+                info['serial_number'] == self.signer_serial_number
+                )
```

Comment:
super small nit, can you indent that parenthesis either after `self.signer_serial_number` or one level less on the same level of `is_signer_certificate`?